internal/controller/spectrumxrailpoolconfig_host_flows_controller.go

```
@@ -311,7 +345,150 @@ func (r *SpectrumXRailPoolConfigHostFlowsReconciler) configureXPlane(ctx context
 		return nil
 	}
 
-	return r.createXPlaneBridges(rt, localNodeState)
+	err := r.createXPlaneBridges(rt, localNodeState)
+	if err != nil {
+		return fmt.Errorf("failed to create X-Plane bridges", err)
```

golangci-lint (2.11.4)

[error] printf: fmt.Errorf call has arguments but no formatting directives

(govet)